examples/dialogs/dialogs/app.py

```diff
@@ -24,11 +24,22 @@ def action_open_file_dialog(self, widget):
         try:
             fname = self.main_window.open_file_dialog(
                 title="Open file with Toga",
+                multiselect=False
             )
             self.label.text = "File to open:" + fname
         except ValueError:
             self.label.text = "Open file dialog was canceled"
 
+    def action_open_file_dialog_multi(self, widget):
```

Comment:
@nehararora did some manual testing and fname is None if the user exits the dialog w/o selecting a file, resulting in a TypeError when concatenating a str with NoneType in line 28. Could we include a ternary to cast fname to empty string if open_file_dialog() returns None, or return empty string within the method itself?

Reply:
@yingw787 good catch! fixed.